src/components/leaflet/index.js

```diff
@@ -93,7 +93,7 @@ var Leaflet = Base.extend({
 
   rotateMarker: function(e) {
     var heading = e.geoposition.coords.heading;
-    heading && this.marker && this.marker.setIconAngle(heading);
+    this.marker && this.marker.setIconAngle(heading);
```

Comment:
you're right, what do you think about something like:

`!isNaN(heading) && heading !== null && this.marker && this.marker.setIconAngle(heading);`

or an explicit `if` is needed in this case ?